pkg/commands/tls/custom/certificate/certificate_test.go

```diff
@@ -255,22 +255,39 @@ func TestTLSCustomCertList(t *testing.T) {
 }
 
 func TestTLSCustomCertUpdate(t *testing.T) {
+	var content string
 	args := testutil.Args
 	scenarios := []testutil.TestScenario{
 		{
 			Name:      "validate missing --cert-blob flag",
 			Args:      args("tls-custom certificate update --id example"),
-			WantError: "required flag --cert-blob not provided",
+			WantError: "neither --cert-path or --cert-blob provided, one must be provided",
```

Comment:
Feels like this test case can be removed in favour of `"validate missing --cert-blob and --cert-path flags"` below.

Reply:
Done!